subcommands/common_config.go

```diff
@@ -93,6 +98,11 @@ func LogConfigs(opts *LogConfigsOptions) {
 		}
 		DieNotNil(err)
 		for _, cfg := range dcl.Configs {
+			if len(cfg.CreatedBy) > 0 {
+				if index := slices.IndexFunc(users, func(user client.FactoryUser) bool { return user.PolisId == cfg.CreatedBy }); index >= 0 {
+					cfg.CreatedBy = users[index].Name
```

Comment:
we might format this like `<id> / <name>` so you get something like `59db9c9a1c85010019e023cc / Andy Doan`